src/components/20-molecules/datepicker/index.js

```diff
@@ -253,6 +254,7 @@ class AXADatepicker extends NoShadowDOM {
     // internal model state
     this.state = {};
     applyDefaults(this);
+    this.handleInputKeyUp = this.handleInputKeyUp.bind(this);
```

Comment:
Is there a reason for two different names?
I personally would name both handleInputKeyUp and onInputKeyUp or handleInputfieldKeyUp and onInputfieldKeyUp